http-add-on/values.yaml

```diff
@@ -0,0 +1,5 @@
+version: v0.1.0-v1alpha1
```

Comment:
Since we have the version here, why do the container images contain a tag? Where is this used then?

Reply:
Indeed, I'm changing these